--- configure.wrf.old	2021-04-23 00:16:14.029041591 +0200
+++ configure.wrf	2021-04-23 00:17:30.890086838 +0200
@@ -116,7 +116,7 @@ LIBWRFLIB = libwrflib.a
                       $(WRF_SRC_ROOT_DIR)/frame/pack_utils.o
 
  LIB_EXTERNAL    = \
-                      -L$(WRF_SRC_ROOT_DIR)/external/io_netcdf -lwrfio_nf -L/usr/lib       
+                      -L$(WRF_SRC_ROOT_DIR)/external/io_netcdf -lwrfio_nf -L/usr/lib -lnetcdf -lnetcdff
 
 
 #### Architecture specific settings ####
@@ -139,11 +139,11 @@ LD              =       $(FC)
 RWORDSIZE       =       $(NATIVE_RWORDSIZE)
 PROMOTION       =       #-fdefault-real-8
 ARCH_LOCAL      =       -DNONSTANDARD_SYSTEM_SUBR  -DWRF_USE_CLM
-CFLAGS_LOCAL    =       -w -O3 -c  # -DRSL0_ONLY
+CFLAGS_LOCAL    =       -w -O3 -c -march=haswell -mtune=generic  # -DRSL0_ONLY
 LDFLAGS_LOCAL   =       
 CPLUSPLUSLIB    =       
 ESMF_LDFLAG     =       $(CPLUSPLUSLIB)
-FCOPTIM         =       -O2 -ftree-vectorize -funroll-loops
+FCOPTIM         =       -O3 -ftree-vectorize -funroll-loops -march=haswell -mtune=generic
 FCREDUCEDOPT	=       $(FCOPTIM)
 FCNOOPT		=       -O0
 FCDEBUG         =       # -g $(FCNOOPT) # -ggdb -fbacktrace -fcheck=bounds,do,mem,pointer -ffpe-trap=invalid,zero,overflow
