--- configure.wps.old	2021-04-23 00:53:36.250906241 +0200
+++ configure.wps	2021-04-23 00:54:41.302817516 +0200
@@ -39,13 +39,14 @@ WRF_INCLUDE     =       -I$(WRF_DIR)/ext
                         -I$(WRF_DIR)/external/io_grib1 \
                         -I$(WRF_DIR)/external/io_int \
                         -I$(WRF_DIR)/inc \
-                        -I$(NETCDF)/include
+                        -I$(NETCDF)/include \
+                        -I/usr/lib64/gfortran/modules
 
 WRF_LIB         =       -L$(WRF_DIR)/external/io_grib1 -lio_grib1 \
                         -L$(WRF_DIR)/external/io_grib_share -lio_grib_share \
                         -L$(WRF_DIR)/external/io_int -lwrfio_int \
                         -L$(WRF_DIR)/external/io_netcdf -lwrfio_nf \
-                        -L$(NETCDF)/lib  -lnetcdf
+                        -L$(NETCDF)/lib  -lnetcdf -lnetcdff -fopenmp
 
 #### Architecture specific settings ####
 
@@ -67,8 +68,8 @@ DM_CC               = mpicc
 FC                  = $(SFC) 
 CC                  = $(SCC)
 LD                  = $(FC)
-FFLAGS              = -ffree-form -O -fconvert=big-endian -frecord-marker=4
-F77FLAGS            = -ffixed-form -O -fconvert=big-endian -frecord-marker=4
+FFLAGS              = -ffree-form -O -fconvert=big-endian -frecord-marker=4 -fallow-argument-mismatch -fallow-invalid-boz
+F77FLAGS            = -ffixed-form -O -fconvert=big-endian -frecord-marker=4 -fallow-argument-mismatch -fallow-invalid-boz
 FCSUFFIX            = 
 FNGFLAGS            = $(FFLAGS)
 LDFLAGS             =
